openedx/adg/lms/webinars/helpers.py

```diff
@@ -83,6 +85,95 @@ def send_webinar_registration_email(webinar, email):
     })
 
 
+def schedule_webinar_reminders(user_email, webinar):
+    """
+    Schedule reminders for a webinar on mandrill.
+
+    Args:
+        user_email (str): User email to schedule a reminder.
+        webinar (Webinar): The webinar for which reminders will be scheduled.
+
+    Returns:
+        None
+    """
+    send_webinar_emails(
+        MandrillClient.WEBINAR_TWO_HOURS_REMINDER,
+        webinar,
+        [user_email],
+        webinar.start_time - timedelta(hours=2),
+    )
+
+    week_before_webinar_start_time = webinar.start_time - timedelta(days=7)
+
+    if week_before_webinar_start_time > now():
```

Comment:
This is intentional. If its less than a week then it will not make sense to send email like "You webinar is one week away". But in case of 2 hours we can send as in that email we are saying that "Your webinar is starting soon". This way the users will get an email in case start time is updated or start time is less than 2 hours.